handbook/company/communications.md

```diff
@@ -213,31 +213,21 @@ Fleet prefixes all GitHub labels with special characters or words to organize an
 
 | Prefix            | Label type  | Examples                            |
 |:------------------|:------------|:------------------------------------|
-| `#`               | Noun        | `#g-marketing`, `#g-ceo`, `#agent`
 | `:`               | Verb        | `:dev`, `:research`, `:design`
 | `~`               | Adjective   | `~blocked`, `~frontend`, `~backend`
 | `customer-`       | [Customer request](TODO link to handbook section) | `customer-leo`, `customer-sagittarius`
 | `#g-`             | Group isssue | _An issue requesting something from a group at Fleet, such that it will be seen and procesed on their kanban board within 1 business day._
 
-Opinionated conventions help people work faster and spend less time figuring out what to name things, or misunderstanding why they're named what they are.  This also reduces the total number of labels required while maintaining an expressive labeling system. For example, instead of a label called `platform-dev-backend`, we use `#platform :dev ~backend`.
+Opinionated conventions help people work faster and spend less time figuring out what to name things, or misunderstanding why they're named what they are.  This also reduces the total number of labels required while maintaining an expressive labeling system.
 
 Labels with a `#g-` prefix refer to a kanban board. Since it is best practice to have an issue on a single board, make an effort to have only one label with the `#g-` prefix per issue.
 
 > _**Note:** There are only a few "special" labels that are exceptions to this rule:
-> - `bug`
-> - `story`
-> -  `prospect-` _(TODO: This makes sense, but I noticed on Nov 30, 2023 that `customer-` was being used in at least one place `prospect-` was supposed to be.  Let's decide what we'll be doing.  Easiest thing is probably to switch to `customer-` for everything and then clarify who is a customer in the mapping between labels and actual organization names.  Up to Luke and Zay.)_
-> - `customer request` _(FUTURE: is this needed?  if so, could use special symbol prefix in front of it)_
```

Comment:
Not needed, changed to `~customer request`

Reply:
@lukeheath all customer requests should have a customer code name label - so possible we don't need this? but up to you

Reply:
That's a good point; seems repetitive. I'll make sure all issues using this have a customer label, and then delete it.

Reply:
@zayhanlon Looks like there are a lot of newer issues using this label with no associated customer label: https://github.com/fleetdm/fleet/labels/~customer%20request. 

I'm going to leave this in place for now, but if we get all issues re-assigned to customer labels we can remove it.

Reply:
Digging deeper and looks like up until two months ago all issues created with "Feature request" had the `~customer request` label applied. Rachael removed that in this PR: https://github.com/fleetdm/fleet/pull/14881/files

Which means there are a lot of issues that are not customer requests using that label. Is it okay for me to remove it en masse, or would you like to review them first?